streams/hk.m3u

```diff
@@ -60,8 +60,6 @@ https://tvp22.sky4k.top/index1.php
 #EXTINF:-1 tvg-id="ViuTV.hk" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36 Edge/12.246",ViuTV (1080p) [Geo-blocked]
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36 Edge/12.246
 https://cdn.hklive.tv/xxxx/99/index.m3u8
-#EXTINF:-1 tvg-id="ViuTV.hk" status="online",ViuTV (720p)
-https://live.astradamy.com/viutv/index.m3u8
```

Comment:
Not broken.

Reply:
Looped video:
<img width="1413" alt="image" src="https://user-images.githubusercontent.com/7253922/232807129-4aef0758-8d3a-48da-b325-9d07d0ed8ad1.png">